account_payment_order/models/account_move.py

```diff
@@ -1,12 +1,139 @@
+# © 2013-2014 ACSONE SA (<https://acsone.eu>).
+# © 2014 Serv. Tecnol. Avanzados - Pedro M. Baeza
 # © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
-from odoo import fields, models
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
 
 
 class AccountMove(models.Model):
     _inherit = "account.move"
 
     payment_order_id = fields.Many2one(
-        "account.payment.order", string="Payment Order", copy=False, readonly=True
+        comodel_name="account.payment.order",
+        string="Payment Order",
+        copy=False,
+        readonly=True,
     )
+    payment_order_ok = fields.Boolean(compute="_compute_payment_order_ok")
+    # we restore this field from <=v11 for now for preserving behavior
+    # TODO: Check if we can remove it and base everything in something at
```

Comment:
Can you check this?

Reply:
There is no field available on invoice / payment mode...

Reply:
OK, I fear this is not correctly implemented as is right now, but I'm not using this kind of structured communication either.